src/Microsoft.SqlTools.ServiceLayer/Profiler/ProfilerService.cs

```diff
@@ -202,6 +204,24 @@ internal int StartSession(string ownerUri, string template, ConnectionInfo connI
             return xeSession.Id;
         }
 
+        private static BaseXEStore CreateXEventStore(ConnectionInfo connInfo, SqlStoreConnection connection)
+        {   
+            BaseXEStore store = null;
```

Comment:
not related to these changes, but I would change `BaseXEStore` to `XEStoreBase`,  putting `Base` at the end makes related types sort better alphabetically if there are several different types of "base" classes in a project.

Reply:
@kburtram I agree, but 'BaseXEStore' comes from 'Microsoft.SqlServer.Management.XEvent.BaseXEStore' so changing this isn't very feasible